### ISSUE TITLE
Hearing List: The list of labels should come after the hearing title

### ISSUE THREAD

**Issue by juhakujala** [City-of-Helsinki/kerrokantasi-ui#699]
The list of labels should be placed after the hearing title. Now screen reader users need to go through all the labels before getting to the main title of the hearing.

If the labels come after the title, they also have a clear context on what hearing they belong to.

Currently on the homepage the labels are placed differently on the hearing cards. They are placed on the bottom of the card which is better for screen reader users.

Current situation:
![Screenshot 2019-06-18 at 10 02 23](https://user-images.githubusercontent.com/7901266/59671297-1ac23c80-91c6-11e9-8b48-281fc21f3776.png)

**Comment by juhakujala** [comment]
Here are some ideas of how the hearing list card design could be changed.

Idea 1:
![Screenshot 2019-06-18 at 11 00 30](https://user-images.githubusercontent.com/7901266/59671562-94f2c100-91c6-11e9-9fea-597d78c0388b.png)

Idea 2:
![Screenshot 2019-06-18 at 11 42 25](https://user-images.githubusercontent.com/7901266/59671577-9c19cf00-91c6-11e9-84ea-e6ef76a95fe1.png)

Idea 3:
![Screenshot 2019-06-18 at 11 49 10](https://user-images.githubusercontent.com/7901266/59671616-a63bcd80-91c6-11e9-8908-b62c6db08a45.png)

Idea 4:
![Screenshot 2019-06-18 at 12 34 01](https://user-images.githubusercontent.com/7901266/59671640-af2c9f00-91c6-11e9-80d1-9ff7d377fc1f.png)

**Comment by juhakujala** [comment]
This was decided that we proceed with the idea #4 

https://user-images.githubusercontent.com/7901266/59671640-af2c9f00-91c6-11e9-80d1-9ff7d377fc1f.png
